openapi-bal-service/src/main/java/io/ballerina/openapi/converter/diagnostic/DiagnosticMessages.java

```diff
@@ -0,0 +1,70 @@
+/*
+ *  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package io.ballerina.openapi.converter.diagnostic;
+
+import io.ballerina.tools.diagnostics.DiagnosticSeverity;
+
+/**
+ * This {@code DiagnosticMessages} enum class for containing the error message related to ballerina to openapi command.
+ *
+ * @since 2.0.0
+ */
+public enum DiagnosticMessages {
+    OAS_CONVERTOR_100("OAS_CONVERTOR_100",
+            "Generated OpenAPI definition does not contain details for the `default` " +
+                    "resource method in the Ballerina service.", DiagnosticSeverity.WARNING),
+    OAS_CONVERTOR_101("OAS_CONVERTOR_101", "Unexpected value: %s", DiagnosticSeverity.ERROR),
+    OAS_CONVERTOR_102("OAS_CONVERTOR_102", "Invalid mediaType : %s", DiagnosticSeverity.ERROR),
+    OAS_CONVERTOR_103("OAS_CONVERTOR_103", "No related status code for : %s", DiagnosticSeverity.ERROR),
+    OAS_CONVERTOR_104("OAS_CONVERTOR_104",
+            "Generated OpenAPI definition does not contain details for the " +
+                    "resource function which has `http:Request` parameters in the Ballerina service.",
+            DiagnosticSeverity.WARNING),
+    OAS_CONVERTOR_105("OAS_CONVERTOR_105",
+            "Generated OpenAPI definition does not contain details for the " +
+                    "resource function which has `http:Response` as return type in the Ballerina service.",
+            DiagnosticSeverity.WARNING),
+    OAS_CONVERTOR_106("OAS_CONVERTOR_106", "Given ballerina file has syntax/compilation error.",
+            DiagnosticSeverity.ERROR),
+    OAS_CONVERTOR_107("OAS_CONVERTOR_107", "No Ballerina services found with name '%s' to" +
+            " generate an OpenAPI specification. These services are available in ballerina file. %s",
+            DiagnosticSeverity.ERROR),
+    OAS_CONVERTOR_108("OAS_CONVERTOR_108", "Having Exception issues : ", DiagnosticSeverity.ERROR);
```

Comment:
WDYM by "having exception issues" ?

Reply:
This is for generic exception handling issues such that when the Path resolving and File writing, reading when java inbuilt functions throwing issues. Instead of saying `Having Exception issues:` shall I keep only "%s" string template there? because the main message will be the message return from `e.getLocalizedMessage()` . 🙂

Reply:
@lnash94 +1 for your suggestion. Also we can modify into something like `Failed to generate OpenAPI definition due to: %s`.  I'm fine with both options